### ISSUE TITLE
Huge ships can have double-recurring charges

### ISSUE THREAD

**Issue by sirjorj** [geordanr/xwing-miniatures-font#85]
![45E0B8C5-92DD-455B-AADF-16811C913EC7](https://user-images.githubusercontent.com/6219508/62645124-3669d980-b911-11e9-80a1-c4dc88d8026f.jpeg)

**Comment by ScottKarch** [comment]
I fixed the force icon/token that was kind of mangled.  And the double force icon is the underscore _ [/underscore]
![image](https://user-images.githubusercontent.com/19825797/67170752-70c40e00-f379-11e9-82bc-30644265e1c2.png)
[xwing-miniatures-SLK-OctFuseForceDouble.ttf.zip](https://github.com/geordanr/xwing-miniatures-font/files/3748741/xwing-miniatures-SLK-OctFuseForceDouble.ttf.zip)

this font also has the fuse I made last night.  I think this clears 3 issues.   double recurring, force icon and fuse.
